### ISSUE TITLE
[MacOS] Module Import error on Scaffold Code

### ISSUE THREAD

**Issue by ganeshmani** [blitz-js/blitz#414]
### What is the problem?

 I have created a scaffold with command `blitz generate recipe`. generated code has module import error for queries and mutation.

<img width="695" alt="Screenshot 2020-05-04 at 12 45 28 PM" src="https://user-images.githubusercontent.com/17050715/80943752-896faf00-8e05-11ea-9d0c-96854df9bd5a.png">

### Versions:

```
checklist-app $ blitz --version --verbose
You are using alpha software - if you have any problems, please open an issue here:
  https://github.com/blitz-js/blitz/issues/new/choose

debug: global
debug: pkgPath: /usr/local/lib/node_modules/blitz/node_modules/@blitzjs/cli/lib/src/index.js 

macOS Catalina | darwin-x64 | Node: v12.16.2

blitz: 0.8.1 (global)
```

### Result

<img width="1118" alt="Screenshot 2020-05-04 at 12 52 54 PM" src="https://user-images.githubusercontent.com/17050715/80944020-30ece180-8e06-11ea-9c2e-cf2d992d82c8.png">

<img width="1118" alt="Screenshot 2020-05-04 at 12 49 03 PM" src="https://user-images.githubusercontent.com/17050715/80943938-0733ba80-8e06-11ea-8082-b673aaed3160.png">

**Comment by merelinguist** [comment]
Thanks for the issue! Could you try:

1. `rm -rf .blitz`

2. `yarn start` again

And see if that makes any difference?

**Comment by ganeshmani** [comment]
you mean `blitz start`. i tried it and it outputs like. 

![Screenshot 2020-05-04 at 1 03 39 PM](https://user-images.githubusercontent.com/17050715/80944665-b4f39900-8e07-11ea-92ef-61a8f300248d.png)

i could solve this issue by importing it like

<img width="571" alt="Screenshot 2020-05-04 at 1 04 46 PM" src="https://user-images.githubusercontent.com/17050715/80944737-e0768380-8e07-11ea-9f30-10690759d592.png">

**Comment by simonpeterdebbarma** [comment]
Hi @ganeshmani, thank you for submitting an issue.

@merelinguist Is this similar to https://github.com/blitz-js/blitz/issues/193? My comment in the aforementioned issue https://github.com/blitz-js/blitz/issues/193#issuecomment-623038968 seems similar, don't it?

**Comment by merelinguist** [comment]
@simonpeterdebbarma Not sure about that, since @ganeshmani is using macOS.

**Comment by simonpeterdebbarma** [comment]
@merelinguist Are the packages different for Mac and Windows?

**Comment by merelinguist** [comment]
Yes. From the looks of it, #193 is related to specific problems related to running `child_process` on Windows. This error looks more like a cross-platform issue.

**Comment by gielcobben** [comment]
When I use `blitz generate all recipe` it created these files:

```
CREATE    app/recipes/pages/recipes/[id]/edit.tsx
CREATE    app/recipes/pages/recipes/[id].tsx
CREATE    app/recipes/pages/recipes/index.tsx
CREATE    app/recipes/pages/recipes/new.tsx
CREATE    app/recipes/queries/recipes/getRecipe.ts
CREATE    app/recipes/queries/recipes/getRecipes.ts
CREATE    app/recipes/mutations/recipes/createRecipe.ts
CREATE    app/recipes/mutations/recipes/deleteRecipe.ts
CREATE    app/recipes/mutations/recipes/updateRecipe.ts
```

that means that the import in your code is wrong. It should be:
```
import getRecipes from app/recipes/queries/recipes/getRecipes.ts
```

**Comment by ganeshmani** [comment]
@gielcobben Got it. i think the scaffold missed the folder structure on import. when i change it, the error is gone.

```
CREATE    app/recipes/pages/recipes/[id]/edit.tsx
CREATE    app/recipes/pages/recipes/[id].tsx
CREATE    app/recipes/pages/recipes/index.tsx
CREATE    app/recipes/pages/recipes/new.tsx
CREATE    app/recipes/queries/recipes/getRecipe.ts
CREATE    app/recipes/queries/recipes/getRecipes.ts
CREATE    app/recipes/mutations/recipes/createRecipe.ts
CREATE    app/recipes/mutations/recipes/deleteRecipe.ts
CREATE    app/recipes/mutations/recipes/updateRecipe.ts
```

**Comment by ganeshmani** [comment]
@simonpeterdebbarma i would like to take up this issue if anybody is not working on it.

**Comment by simonpeterdebbarma** [comment]
Sure, go ahead. Thank you very much!

**Comment by flybayer** [comment]
Hey y'all! We are tracking this issue here: https://github.com/blitz-js/blitz/issues/402

Should have a hot fix soon.

**Comment by flybayer** [comment]
This is fixed in blitz `0.8.2`!
